Podfile

```diff
@@ -47,7 +47,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-    pod 'WordPressKit', '~> 4.39.0'
+    pod 'WordPressKit', '~> 4.40.0-beta'
```

Comment:
Was this supposed to be part of this PR?  I'm missing how WordPressKit would be used.

Reply:
Oh. I see that it's used in https://github.com/wordpress-mobile/WordPress-iOS/pull/17006 and that PR is both branched from and targets this one. Let's just roll with it. :)